tests/phpunit/integration/tests/REST_API/Stories_Controller.php

```diff
@@ -533,22 +533,71 @@ public function test_get_item_with_featured_image(): void {
 			]
 		);
 
-		$attachment_id = self::factory()->attachment->create_upload_object( WEB_STORIES_TEST_DATA_DIR . '/attachment.jpg', 0 );
+		$attachment_id = self::factory()->attachment->create_upload_object( WEB_STORIES_TEST_DATA_DIR . '/example-3.png', 0 );
+		wp_maybe_generate_attachment_metadata( get_post( $attachment_id ) );
 		set_post_thumbnail( $story, $attachment_id );
 
-		$attachment_src = wp_get_attachment_image_src( $attachment_id, Image_Sizes::POSTER_PORTRAIT_IMAGE_SIZE );
+		$attachment_src = wp_get_attachment_image_src( $attachment_id, Image_Sizes::POSTER_PORTRAIT_IMAGE_DIMENSIONS );
 
 		$request  = new WP_REST_Request( \WP_REST_Server::READABLE, '/web-stories/v1/web-story/' . $story );
 		$response = rest_get_server()->dispatch( $request );
 		$data     = $response->get_data();
 
 		$this->assertArrayHasKey( 'story_poster', $data );
+		$this->assertSame( Image_Sizes::POSTER_PORTRAIT_IMAGE_DIMENSIONS[0], $attachment_src[1] );
+		$this->assertSame( Image_Sizes::POSTER_PORTRAIT_IMAGE_DIMENSIONS[1], $attachment_src[2] );
 		$this->assertEqualSetsWithIndex(
 			[
 				'id'         => $attachment_id,
 				'url'        => $attachment_src[0],
-				'height'     => $attachment_src[1],
-				'width'      => $attachment_src[2],
+				'height'     => $attachment_src[2],
```

Comment:
Nit: can we flip these so it's width before height 

width [1]
width [2]

Little better readability IMO